src/toil/common.py

```diff
@@ -517,6 +517,9 @@ def addOptions(parser: ArgumentParser, config: Config = Config()):
     resource_options.add_argument('--defaultDisk', dest='defaultDisk', default=None, metavar='INT',
                                   help=resource_help_msg.format('default', 'disk', disk_mem_note,
                                                                 bytes2human(config.defaultDisk, symbols="iec")))
+    assert not config.defaultPreemptable, 'User would be unable to reset config.defaultPreemptable'
```

Comment:
I think we're trying to stop someone setting this in the workflow script, because it would make their script UI bad.

I'll make it work like `--disableCaching`, where you can pass a True or False if you want to.